quickstart-download.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Downloading openshift quickstart examples
+  hosts: localhost
+  vars_files:
+    - config.yml
+  environment:
+    KUBECONFIG: "{{ kubeconfig }}"
+  roles:
+    - login_ocp
+  post_tasks:
```

Comment:
It may be better to pass template url directly to ```oc create -f ``` instead of downloading templates

Reply:
That is true, didn't noticed that it works.